frontend/src/Pages/ProgramOverviewDashboard.tsx

```diff
@@ -330,104 +326,126 @@ export default function ProgramOverviewDashboard() {
-                                                            program_class.id
-                                                        )
-                                                    }
-                                                />
-                                            )}
-                                            <ULIComponent
-                                                dataTip={'Class Overview'}
-                                                icon={PuzzlePieceIcon}
-                                                iconClassName={
-                                                    '!w-6 !h-6 cursor-pointer hover:text-primary hover:scale-110 transition-all'
-                                                }
-                                                onClick={() => {
-                                                    navigate(
-                                                        `/program-classes/${program_class.id}/dashboard`
-                                                    );
-                                                }}
-                                            />
-                                        </div>
+                                    <td
+                                        onClick={(e) => e.stopPropagation()}
+                                        className="px-4 py-2 text-left"
+                                    >
+                                        <input
+                                            type="checkbox"
+                                            className={`cursor-pointer checkbox checkbox-sm ${
+                                                isArchived(pc)
+                                                    ? 'invisible'
+                                                    : ''
+                                            }`}
+                                            checked={isSelected}
+                                            disabled={isArchived(pc)}
+                                            onChange={() =>
+                                                handleToggleRow(pc.id)
+                                            }
+                                        />
                                     </td>
-                                    <td>
-                                        <ClampedText as="div" lines={1}>
-                                            {program_class.name}
+
+                                    <td
+                                        className="px-4 py-2 cursor-pointer text-center"
+                                        onClick={() =>
+                                            navigate(
+                                                `/program-classes/${pc.id}/dashboard`
+                                            )
+                                        }
+                                    >
+                                        <ClampedText
```

Comment:
@corypride Can we apply an undeline on hover of the cell instead so that the user knows it'll be clickable?